web/templatetags/custom_filters.py

```
@@ -0,0 +1,9 @@
+# In your_app/templatetags/custom_filters.py
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def multiply(value, arg):
```

Ruff (0.8.2)

Missing return type annotation for public function `multiply`

(ANN201)